modules/OAuth2Clients/language/en_us.lang.php

```diff
@@ -49,6 +49,6 @@
     'LBL_IS_CONFIDENTIAL'=> 'Is confidential',
     'LBL_SECRET'=> 'Secret',
 
-    'LNK_OAUTH2_CLIENT_LIST'=> 'List OAuth2Clients',
-    'LNK_NEW_OAUTH2_CLIENT'=> 'Create OAuth2Client',
+    'LNK_OAUTH2_CLIENT_LIST'=> 'List OAuth2 Clients',
```

Comment:
Shouldn't also be change the other strings module name and title for this module (see lines above)?

Reply:
Absolutely correct! Thanks

Reply:
reverted in https://github.com/salesagility/SuiteCRM/pull/5180/files/ed3f79bd4ff065d0f60fb98bb8ef6eac46406512..670dfda30e0f4d735106a321b5da65ee763f9913 ?

Reply:
Yes, turns out it should to be OAuth2. This is intentional.